### ISSUE TITLE
default profile ignored

### ISSUE THREAD

**Issue by mciureanu** [aws/aws-sdk-net#1257]
Whether I'm using dotnet lambda, or dotnet test, the default profile gets completly ignored.  If I specify a named profile in appsettings, things work ok, or if I use --profile in dotnet lambda command (like package-ci).  If I use "default" profile like that, things don't work.

  "AWS": {
    "Region": "ap-southeast-1",
    "Profile":  "default"
  },

I installed aws cli with python, and something changed at this point, but not sure what. The credentials in .aws folder look ok. 

Not sure I'm looking in the right place - the dotnet sdk reads the profile information directly from the files (credentials and config) ?

**Comment by vellozzi** [comment]
Are you running on a machine with AWS Toolkit for Visual Studio installed?  If so, it may also be looking at C:\Users\<username>\AppData\Local\AWSToolkit\RegisteredAccounts.json .  RegisteredAccounts.json takes percedence over config and credentials.  Let us know if that's the situation.

**Comment by mciureanu** [comment]
Thank you very much. I think that was the issue. I didn't install AWS Toolkit for Visual Studio, and I didn't have Visual Studio, but I installed something else, I think - AWS Tools for PowerShell. I uninstalled this, but the file was not deleted.

I was surprised that this precedence is embedded in the aws.sdk. I deleted the AWSTookit folder and everything works ok now. (I had to debug the sdk to get to this conclusion, quite a pain :) )

Thank you for the comment.
